packer/variables.pkr.hcl

```diff
@@ -22,13 +24,13 @@ variable "aws_region" {
 
 variable "aws_source_ami_filter_name" {
   type        = string
-  default     = "CentOS 8* x86_64*"
+  default     = "RHEL-6.10*"
```

Comment:
This will need to change to match the format of the AMI Name we created as part of the base AMI.

Reply:
rhel6-base*